Gemfile

```diff
@@ -36,6 +36,7 @@ group :development, :test do
   gem 'factory_girl_rails'
   gem 'faker'
   gem 'awesome_print', '~> 1.8'
+  gem 'pry-rails'
```

Comment:
This gem does a lot of cool stuff.  The reason I am adding it here is because, by default, it pretty prints things in the `rails console`.  So for example, when you are viewing a deeply nested array of hashes, instead of printing it out all in one line, it will add in the proper indenting and line returns.  

I used it to get the shape of the array in the new `group_endpoint_parsed_response` method.